src/PokeList.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: academy
+ * Date: 14/11/2018
+ * Time: 11:48
+ */
+
+namespace Pokedex;
+
+use Pokedex\DbConnection;
```

Comment:
This is unused on this page. You have 'used' in index.php which allows it to work.

Reply:
removed in 6370f38